processor/spanmetricsprocessor/config.go

```diff
@@ -86,6 +86,39 @@ type Config struct {
 	// LaunchDarklyKey defines the LaunchDarkly key.
 	// Optional. Only required when `EnableFeatureFlag` is `true`.
 	LaunchDarklyKey string `mapstructure:"launch_darkly_key"`
+
+	// Transforms takes in a list of Actions to perform on metrics generated by span metrics processor.
+	// currently there is only one action, the rename action. This allows a user to specify metrics with certain attributes
+	// to have a specific metric name. By default, the generated metric names for Rate / Error is "calls_total" and Duration is
+	// "latency".
+	Transforms []Transform `mapstructure:"transforms"`
```

Comment:
How about just calling this `Renames []Rename`? The struct itself is dedicated to renaming transform already.